docker/server/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker build -t turbinia-server:$1 .
+docker tag turbinia-server:$1  gcr.io/oss-forensics-registry/turbinia/turbinia-server:$1
```

Comment:
This is going to change when we setup the build pipeline. But fine to leave it like this for now.